test/contrib/bigquery_test.py

```diff
@@ -147,3 +149,31 @@ def output(self):
             }
         }
         run_job.assert_called_with('proj', expected_body, dataset=BQDataset('proj', 'ds', None))
+
+
+class BigQueryClientTest(unittest.TestCase):
+
+    def test_retry_succeeds_on_second_attempt(self):
+        try:
+            from googleapiclient import errors
+        except ImportError:
+            raise unittest.SkipTest('Unable to load googleapiclient module')
+        client = MagicMock(spec=bigquery.BigQueryClient)
```

Comment:
Is this qualified name required because of mocking? Can we do
```python
from luigi.contrib.bigquery import BigQueryClient, BigQueryLoadTask, BigQueryTarget, BQDataset, \
    BigQueryRunQueryTask, BigQueryExtractTask
```